### PR TITLE
FOGL-7509: greyed out menu items when fledge is down

### PR DESCRIPTION
**Pull request by mshariq-nerd**

### CODE REVIEW

**Review by monika-Dianomic — DISMISSED** [review]

**Review by praveen-garg — DISMISSED** [review]
We can probably add opacity to container and also the open modal if any.  
We should not close the modal though as it can be temporarily down and may have to attempt 3 times if want to close the modal.

File: src/app/components/layout/navbar/navbar.component.css [hunk 1]
```diff
@@ -86,6 +86,13 @@
...
-  cursor: default;
-}
+.disabled {
+  opacity: .95;
+  background-color: whitesmoke;
+  pointer-events: none;
+ }
+
+ .help {
+  margin-top: 0px;
```
Comment (praveen-garg) [inline src/app/components/layout/navbar/navbar.component.css:96]: Not for any with help class but for this ID only?!

Comment (mshariq-nerd) [reply src/app/components/layout/navbar/navbar.component.css:96]: help class is used only in the span that is to show the Auto refresh not enable message in navbar.

**Review by hemantgarg923 — APPROVED** [review]

File: src/app/app.component.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.c1 {
```
Comment (monika-Dianomic) [inline src/app/app.component.css:1]: I think, class name should be meaningful.

**Review by monika-Dianomic — APPROVED** [review]

**Pull request merged**